apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPage.scss

```diff
@@ -40,8 +44,9 @@
             p {
                 font-size: 0.875rem;
                 line-height: 1.4rem;
-                margin-top: 0.5rem;
-                color: colors.$base-light;
+                margin-top: 1rem;
+                margin-bottom: 0.5rem;
+                color: colors.$base-light;-size: 0.75rem;
```

Comment:
Suggestion (non-blocking): was something left out? I see a dangling `-size: 0.75 rem;`

Reply:
fixed it